x-pack/plugins/apm/public/hooks/use_dynamic_index_pattern.ts

```diff
@@ -6,25 +6,14 @@
  */
 
 import { useFetcher } from './use_fetcher';
-import { UIProcessorEvent } from '../../common/processor_event';
 
-export function useDynamicIndexPatternFetcher(
-  processorEvent: UIProcessorEvent | undefined
-) {
-  const { data, status } = useFetcher(
-    (callApmApi) => {
-      return callApmApi({
-        endpoint: 'GET /api/apm/index_pattern/dynamic',
-        isCachable: true,
-        params: {
-          query: {
-            processorEvent,
-          },
-        },
-      });
-    },
-    [processorEvent]
-  );
+export function useDynamicIndexPatternFetcher() {
+  const { data, status } = useFetcher((callApmApi) => {
+    return callApmApi({
+      endpoint: 'GET /api/apm/index_pattern/dynamic',
+      isCachable: true,
+    });
+  }, []);
```

Comment:
Another benefit of removing `processorEvent`: a dynamic index pattern is now only fetched once.